tests/ci/env_helper.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+CI = bool(os.getenv("CI"))
+TEMP_PATH = os.getenv("TEMP_PATH", os.path.abspath("."))
+
+CACHES_PATH = os.getenv("CACHES_PATH", TEMP_PATH)
+CLOUDFLARE_TOKEN = os.getenv("CLOUDFLARE_TOKEN")
+GITHUB_EVENT_PATH = os.getenv("GITHUB_EVENT_PATH")
+GITHUB_REPOSITORY = os.getenv("GITHUB_REPOSITORY", "ClickHouse/ClickHouse")
+GITHUB_RUN_ID = os.getenv("GITHUB_RUN_ID")
+GITHUB_SERVER_URL = os.getenv("GITHUB_SERVER_URL", "https://github.com")
+GITHUB_WORKSPACE = os.getenv("GITHUB_WORKSPACE", os.path.abspath("../../"))
+IMAGES_PATH = os.getenv("IMAGES_PATH")
+REPORTS_PATH = os.getenv("REPORTS_PATH", "./reports")
```

Comment:
Frankly speaking, I don't like such things. When I see `open(REPORTS_PATH) ...` it tells me nothing. Where does this variable come from, how it is initialized? It may sound strange but I'm a big fan of copy-paste for such short parts of logic. Because when you see `os.getenv("REPORTS_PATH", "./reports")` in the beginning of `__main__` it's 100% clear what is going on. But I'm a bad python programmer, so let's try to use them in this way.

Reply:
Hmm, I can replace them back but I think it's not that it really matters here.
My idea was to check if all these are not duplicated too much between different test parts.